flink-mesos/src/main/java/org/apache/flink/mesos/runtime/clusterframework/MesosResourceManagerFactory.java

```diff
@@ -99,4 +100,11 @@ public MesosResourceManagerFactory(@Nonnull MesosServices mesosServices, @Nonnul
 			webInterfaceUrl,
 			jobManagerMetricGroup);
 	}
+
+	@Override
+	public Configuration getResourceManagerConfiguration(Configuration flinkConfig) {
+		final Configuration config = new Configuration(flinkConfig);
+		config.setString(TaskManagerOptions.MANAGED_MEMORY_SIZE, flinkConfig.getInteger(MesosTaskManagerParameters.MESOS_RM_TASKS_MEMORY_MB) + "m");
```

Comment:
Isn't `MESOS_RM_TASKS_MEMORY_MB` the memory a Mesos task should get assigned (aka the total container memory size)? Why did we introduce this change in this PR? It seems a bit unrelated.